tests/frameworks/OSXShims/include/windows.h

```diff
@@ -25,6 +25,10 @@
 #define _MAX_PATH PATH_MAX
 #endif
 
+#ifndef MAX_PATH
+#define MAX_PATH PATH_MAX
```

Comment:
Yeah: `MAX_PATH` lives in `minwindef` standalone and `_MAX_PATH` lives in stdlib with proximity to the standard library path functions.